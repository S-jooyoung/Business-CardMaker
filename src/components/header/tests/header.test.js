// import dependencies
import React from "react";

// import custom-component
import Header from "../header";

// import snapshot-testing methods
import renderer from "react-test-renderer";

describe("Header", () => {
  let headerComponent;
  let onLogout;

  beforeEach(() => {
    onLogout = jest.fn();
    headerComponent = <Header onLogout={onLogout} />;
  });

  it("render", () => {
    const component = renderer.create(headerComponent).toJSON();
    expect(component).toMatchSnapshot();
  });
});
