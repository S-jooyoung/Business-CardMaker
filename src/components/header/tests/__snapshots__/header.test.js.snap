// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render 1`] = `
<header
  className="header"
>
  <button
    className="logout"
    onClick={[MockFunction]}
  >
    Logout
  </button>
  <img
    alt="logo"
    className="logo"
    src="/images/logo.png"
  />
  <h1
    className="title"
  >
    Business Card Maker
  </h1>
</header>
`;
